bundles/statistics/statsgrid2016/FlyoutManager.js

```diff
@@ -3,6 +3,7 @@ Oskari.clazz.define('Oskari.statistics.statsgrid.FlyoutManager', function (insta
     this.flyouts = {};
     var loc = instance.getLocalization();
     Oskari.makeObservable(this);
+    this.service = this.instance.getSandbox().getService('Oskari.statistics.statsgrid.StatisticsService');
```

Comment:
Change this to instance.getStatisticsService().getStateService() so it's clearer that's the one needed.

Reply:
This change is now included to PR.